app/src/main/java/com/greenfox/kalendaryo/SelectCalendarActivity.java

```diff
@@ -87,23 +96,50 @@ public void getCalendarList() {
         ArrayList<String> accounts = kalPref.getAccounts();
 
         for (int i = 0; i < accounts.size(); i++) {
+            CURRENT_ATTEMPT = FIRST_ATTEMPT;
```

Comment:
the value of `CURRENT_ATTEMPT` won't be valid,
let's say you have 3 accounts and want to query their calendars
1. For the 1st account, you set `CURRENT_ATTEMPT` to be `1`
2. You send a request to Google to get the calendars  -> this is an async request, you won't wait for the result just register the behavior that should happen onResponse or onFailure case
3. Program execution goes back to get the next account
4. You send a request to get the calendars for the 2nd account
...etc

You can't be sure that the response will arrive before you set the `CURRENT_ATTEMPT` to 1 again.